contracts/SmartLoan.sol

```diff
@@ -30,10 +30,11 @@ contract SmartLoan is Ownable {
 
   uint256 public minSolvencyRatio = 1200;
 
-  constructor(IPriceProvider _priceProvider, IAssetsExchange _assetsExchange, Pool _pool) {
-    priceProvider = _priceProvider;
-    exchange = _assetsExchange;
-    pool = _pool;
+  function initialize(IPriceProvider priceProvider_, IAssetsExchange assetsExchange_, Pool pool_) external initializer {
+    priceProvider = priceProvider_;
+    exchange = assetsExchange_;
+    pool = pool_;
+    __Ownable_init();
```

Comment:
Hmm, if we init the owner here, maybe there is no need to use the PermissiveOwnable and we can simply use the https://github.com/OpenZeppelin/openzeppelin-contracts-upgradeable/blob/master/contracts/access/OwnableUpgradeable.sol ?

Reply:
I'm also in favour of this approach